scripts/config/tcl.py

```diff
@@ -254,10 +254,15 @@ def process_string(value: str, state: State) -> str:
             full_abspath = os.path.abspath(value)
 
             # Resolve globs for paths that are inside the exposed directory
-            if value.startswith("/") and full_abspath.startswith(found):
-                files = glob.glob(full_abspath)
-                files_escaped = [file.replace("$", r"\$") for file in files]
-                value = " ".join(files_escaped)
+            if value.startswith("/"):
+                if full_abspath.startswith(found):
+                    files = glob.glob(full_abspath)
+                    if not files:
+                        raise InvalidConfig(f"No such file {full_abspath}")
+                    files_escaped = [file.replace("$", r"\$") for file in files]
+                    value = " ".join(files_escaped)
+                else:
+                    raise InvalidConfig(f"Invalid path: {full_abspath}")
```

Comment:
That's not an invalid path... That's literally any path that would ascend from the PDK or SCL.